metron-analytics/metron-profiler-spark/README.md

```diff
@@ -265,6 +290,18 @@ The path to the input data read by the Batch Profiler.
 
 The format of the input data read by the Batch Profiler.
 
+### `profiler.batch.input.reader`
```

Comment:
This feels somewhat unnecessary as a parameter.

Why could we just keep `profiler.batch.input.format` and then drop this?  We can determine reader based on the format (COLUMNAR if ORC/Parquet, TEXT else).  If we add other formats in the future, we'd still know right which reader to pull right?

Reply:
It is a valid option.  The only reason I did not do that is that we would have to specifically support each format like JSON, CSV, ORC, Parquet.  Whereas with these two switches, via configuration alone, a user can use a variety of formats without us having to specifically support each one.

Now that being said, I don't know how useful that is to the user population.  How many formats will users want to consume?  How useful is that flexibility? 

At this point, since this is new functionality, I decided to err on the side of greater flexibility over simplicity. Knowing that reasoning, let me know if you still think we should go for simplicity over flexibility.

Reply:
The main thing I'm getting at here is what happens if instead of 
```
profiler.batch.input.reader=COLUMNAR
profiler.batch.input.format=org.apache.spark.sql.execution.datasources.orc
```

I say
```
profiler.batch.input.reader=TEXT
profiler.batch.input.format=org.apache.spark.sql.execution.datasources.orc
```

Correct me if I'm wrong, but I believe it'll instantiate a `TextEncodedTelemetryReader` instead of a `ColumnEncodedTelemetryReader`, then fail to read the file.

This is a super easy misconfiguration to make as it is right now.  Is it potentially reasonable to keep both fields, but let you shortcut known formats (e.g. ORC and Parquet)?  Or log a warning that a known misconfig happened and then proceed with the COLUMNAR option anyway?

Reply:
@justinleet  I added some configuration shortcuts that make it simpler and less error prone to configure, but still leaves the flexibility there.  I like how this turned out.  

Check-out the README for more examples, but reading ORC now just becomes this.
```
profiler.batch.input.reader=orc
profiler.batch.input.path=/path/to/orc/
```

Thanks for the feedback and let me know if this covers your concerns.